[2023-08-28T05:42:35.339+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tbf_analysis_dag_v20.authenticate_service_acct_task manual__2023-08-28T05:42:32.454625+00:00 [queued]>
[2023-08-28T05:42:35.347+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tbf_analysis_dag_v20.authenticate_service_acct_task manual__2023-08-28T05:42:32.454625+00:00 [queued]>
[2023-08-28T05:42:35.349+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-28T05:42:35.375+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): authenticate_service_acct_task> on 2023-08-28 05:42:32.454625+00:00
[2023-08-28T05:42:35.382+0000] {standard_task_runner.py:57} INFO - Started process 82 to run task
[2023-08-28T05:42:35.390+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'tbf_analysis_dag_v20', 'authenticate_service_acct_task', 'manual__2023-08-28T05:42:32.454625+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/tbf_analysis_dag_v20.py', '--cfg-path', '/tmp/tmpqyysqbp7']
[2023-08-28T05:42:35.403+0000] {standard_task_runner.py:85} INFO - Job 32: Subtask authenticate_service_acct_task
[2023-08-28T05:42:35.435+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-08-28T05:42:35.490+0000] {task_command.py:415} INFO - Running <TaskInstance: tbf_analysis_dag_v20.authenticate_service_acct_task manual__2023-08-28T05:42:32.454625+00:00 [running]> on host 7d623e02890b
[2023-08-28T05:42:35.580+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tbf_analysis_dag_v20' AIRFLOW_CTX_TASK_ID='authenticate_service_acct_task' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T05:42:32.454625+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-28T05:42:32.454625+00:00'
[2023-08-28T05:42:35.583+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-28T05:42:35.586+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo starting_gcloud_auth; gcloud auth activate-service-account terraform@trans-border-freights-394419.iam.gserviceaccount.com --key-file /.google/credentials/google_credentials.json; echo ending_gcloud_auth']
[2023-08-28T05:42:35.603+0000] {subprocess.py:86} INFO - Output:
[2023-08-28T05:42:35.604+0000] {subprocess.py:93} INFO - starting_gcloud_auth
[2023-08-28T05:42:36.355+0000] {subprocess.py:93} INFO - Activated service account credentials for: [terraform@trans-border-freights-394419.iam.gserviceaccount.com]
[2023-08-28T05:42:36.867+0000] {subprocess.py:93} INFO - ending_gcloud_auth
[2023-08-28T05:42:36.868+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-28T05:42:36.892+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=tbf_analysis_dag_v20, task_id=authenticate_service_acct_task, execution_date=20230828T054232, start_date=20230828T054235, end_date=20230828T054236
[2023-08-28T05:42:36.947+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-28T05:42:36.968+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
